eng/Generate.ps1

```diff
@@ -50,6 +51,16 @@ function Add-TestServer-Swagger ([string]$testName, [string]$projectSuffix, [str
     Add-Swagger "$testName$projectSuffix" $projectDirectory "--require=$configurationPath --try-require=$inputReadme --input-file=$inputFile $additionalArgs"
 }
 
+function Add-CadlRanch-Cadl([string]$testName, [string]$cadlRanchDirectory) {
+    $projectDirectory = Join-Path $cadlRanchDirectory $testName
+    $cadlFolders = Get-ChildItem -Path $cadlRanchPath -Depth 2 -Directory $testName
+    if ($cadlFolders) {
+        $cadlFolder = $cadlFolders[0]
+        $cadlMain = Join-Path $cadlFolder "main.cadl"
```

Comment:
seems the $cadlFolder is the same as $projectDirectory. Why need to retrieve again?

Reply:
No, they are different. $projectDirectory is the output folder (test\CadlRanchProjects). $cadlFolder is the cadl filer folder (node_modules\@azure-tools\cadl-ranch-specs\http\resiliency\dev-driven for example)

Reply:
yes. I see. I mixed $cadlRanchDirectory with $cadlRanchPath. It is better to change the '$cadlRanchDirectory' to '$cadlRanchTestProjectDirectory'. In that way, it will more clear.

Reply:
I will change to `$cadlRanchProjectsDirectory`